Sources/MountebankSwift/Models/Stub/Response/Body.swift

```diff
@@ -1,12 +1,14 @@
 import Foundation
 
-private var jsonEncoder = JSONEncoder()
-
 /// A body for an ``Is`` response
 public enum Body: Equatable {
     case text(String)
+    /// A JSON body that will be encoded to a text response
     case json(JSON)
-    case jsonEncodable(Encodable)
+    /// An Encodable body that will be encoded to a text response
+    /// Custom encoding strategies are supported by providing a custom JSONEncoder
+    case jsonEncodable(Encodable, JSONEncoder? = nil)
+    /// A Data response that will be base64 encoded
```

Comment:
It will only `base64` the data for mountebank. Mountebank it self will return it back as data.

Reply:
But this will be part of a bigger imposter, and the data in here will be base64 encoded before it will be send as plain (json) text to mountebank right? 

The JSONEncoder will use the `JSONEncoder.DataEncodingStrategy.base64`, see https://developer.apple.com/documentation/foundation/jsonencoder/2895381-dataencodingstrategy